src/components/paragraphs/stanford-card/card-paragraph.tsx

```diff
@@ -14,6 +14,23 @@ type Props = HtmlHTMLAttributes<HTMLDivElement> & {
   paragraph: ParagraphStanfordCard
 }
 
+const adjustHeadingType = (size: "larger" | "smaller" | "default" | undefined, heading: string): string => {
+  const headingTypes: {[key: string]: number} = {
+    h2: 4,
+    h3: 3,
+    h4: 2,
+  }
+  if (!headingTypes[heading] || !size || size === "default") return ""
+  let type = headingTypes[heading]
+  if (size === "larger") {
+    type += 1
+  } else if (size === "smaller") {
+    type -= 1
+  }
+
+  return `type-${type}`
```

Comment:
remember, dynamic classes aren't reliable. use `clsx` here like 
```
clsx({
  "type-3": type === 3,
  "type-2": type === 2,
})
```

Optionally, we could even rewrite this entire logic to be just a bigger clsx call:

```
const isH2 = heading === "h2"
const isH3 = heading === "h3"
const isH4 = heading === "h4"
const isLarger = size === "larger"
const isSmaller = size === "smaller"

clsx({
  "type-3": (isH2 && !size) || (isH3 && isLarger) ... etc
  "type-2": (isH3 && !size) || (isH2 && isSmaller) ... etc
  "type-1": (isH4 && !size) .... etc
})
```